scripts/format

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${IOW_BOUNDARY_TOOL_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n "Usage: $(basename "$0")
+
+Run formatters on the project's code
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        # Format Django app
+
+        # Get a list of changed ".py" files and trim "src/django"
+        changed_files=$(git status --porcelain | grep ".py" | awk -F'src/django/' '{print $2}' | xargs)
+        if [[ -n "${changed_files}" ]]; then
+            # Run isort --check-only before fixing to ensure a failure code is emitted
+            if ! docker-compose \
+                run --rm --no-TTY --no-deps --entrypoint isort django "${changed_files}" --check-only; then
```

Comment:
Since this is now a list of multiple files, I don't think this should be quoted anymore. Getting this error:

```
isort.exceptions.InvalidSettingsPath: isort was told to use the settings_path: /usr/local/src/api/models/boundary.py api/serializers/boundary.py api/views as the base directory or file that represents the starting point of config file discovery, but it does not exist.
```

Removing the quotes everywhere fixes the run.

Reply:
Ah, then I will have to silence the `shellcheck` warning, which was failing the CI run. I'll look into if they have pragmas for that.